spec/lib/external_count_fetcher_spec.rb

```diff
@@ -2,7 +2,7 @@
 
 describe ExternalCountFetcher do
   before(:all) do
-    @fake_counts = {:supporter_count=>2812, :volunteer_count=>43949, :called_voters_count=>0, :reps_calls_count=>0, :house_supporters=>0, :senate_supporters=>0, :donations_total=>748608206, :donations_count=>65136}
+    @fake_counts = {:supporter_count=>69087, :volunteer_count=>2151, :called_voters_count=>0, :reps_calls_count=>0, :house_supporters=>0, :senate_supporters=>0, :donations_total=>748608206, :donations_count=>65136}
```

Comment:
Line is too long. [214/80]<br>Use the new Ruby 1.9 hash syntax.<br>Surrounding space missing for operator '=>'.<br>Space inside { missing.<br>Space inside } missing.